### DISCUSSION TITLE
blog/advanced-react-patterns

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#400] [General]
# blog/advanced-react-patterns

List of react advanced patterns complete with examples.

https://theodorusclarence.com/blog/advanced-react-patterns

**Comment by ahmad-reza619** [comment]
Nice article! really enjoyed reading this.

However i think the `Function as Child` pattern is more commonly known as render props. since that is what [old react docs named it](https://legacy.reactjs.org/docs/render-props.html)

**Reply by theodorusclarence** [reply]
yep pretty much the same thing, but function as child uses the `children` props, and sugar-syntaxed using jsx.

render props are more specific to the rest of the props, I've seen it used on Table like
```tsx
<Table rowRender={props => <tr>..</tr>} />
```

**Comment by Mukti102** [comment]
Hello
